boards/BMS/Src/Io/Io_Airs.c

```diff
@@ -1,35 +1,51 @@
 #include "Io_Airs.h"
 #include "main.h"
 
-// Since there are no pins to determine the AIRs' statuses directly, the AIRs'
-// statuses can be determined by the LUT below:
-//
-// +------------------+------------------+------------------------+
-// | AIR POWER STATUS | AIR POWER ISENSE |      AIRs statuses     |
-// +------------------+------------------+------------------------+
-// |                0 |                0 | AIR- OPEN, AIR+ OPEN   |
-// |                0 |                1 | NOT POSSIBLE           |
-// |                1 |                0 | AIR- CLOSED, AIR+ OPEN |
-// |                1 |                1 | AIR- CLOSED, AIR+ OPEN |
-// +------------------+------------------+------------------------+
+// Shunt resistor used to measure AIR loop current
+#define AIR_TOTAL_ISENSE_SHUNT_R_OHMS (0.075f)
+
+// Amplifier used to amplify voltage drop across the shunt resistor
+#define AIR_TOTAL_ISENSE_AMPLIFIER (20.0f)
+
+// Converts voltage measured across shunt resistor to current in A
+#define AIR_TOTAL_ISENSE_V_TO_A \
+    (1.0f / (AIR_TOTAL_ISENSE_SHUNT_R_OHMS * AIR_TOTAL_ISENSE_AMPLIFIER))
+
+// AIR positive closed threshold
+#define AIR_POSITIVE_CLOSED_THRESH_A (0.5f)
+
+static bool is_air_positive_closed = false;
 
 bool Io_Airs_IsAirNegativeClosed(void)
 {
     return HAL_GPIO_ReadPin(AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin);
 }
 
+#include "Io_Adc.h"
 bool Io_Airs_IsAirPositiveClosed(void)
 {
-    return HAL_GPIO_ReadPin(AIR_TOTAL_ISENSE_GPIO_Port, AIR_TOTAL_ISENSE_Pin) &&
-           HAL_GPIO_ReadPin(AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin);
+    // const float air_loop_current =
+    //    (float)Io_Adc_GetAdc2Channel4Voltage() * AIR_TOTAL_ISENSE_V_TO_A;
+    // return HAL_GPIO_ReadPin(AIR_POWER_STATUS_GPIO_Port, AIR_POWER_STATUS_Pin)
+    // &&
+    //       (air_loop_current > AIR_POSITIVE_CLOSED_THRESH_A);
+
```

Comment:
i wanted to determine what this current would be. we should be able to get rid of is_air_positive_closed when we merge this in